src/app/components/community/sidebar/sidebar.ts

```diff
@@ -79,35 +61,37 @@ export default class AppCommunitySidebar extends Vue {
 	}
 
 	get shouldShowKnownMembers() {
-		return !!this.app.user && this.knownMembers && this.knownMembers.length > 0;
+		return !!this.app.user && this.data.knownMembers && this.data.knownMembers.length > 0;
 	}
 
 	get membersYouKnowCount() {
-		return number(this.knownMemberCount);
+		return number(this.data.knownMemberCount);
 	}
 
 	get shareUrl() {
-		return Environment.baseUrl + this.$router.resolve(this.community.routeLocation).href;
+		return Environment.baseUrl + this.$router.resolve(this.data.community.routeLocation).href;
 	}
 
 	get shareContent() {
 		return this.$gettextInterpolate('Check out %{ name } community - Game Jolt', {
-			name: this.community.name,
+			name: this.data.community.name,
 		});
 	}
 
 	get hasMoreCollaborators() {
-		return this.currentCollaboratorCount > this.initialCollaboratorCount;
+		return this.currentCollaboratorCount > this.data.initialCollaboratorCount;
 	}
 
 	get moderators(): User[] {
 		const mods = [];
-		if (this.owner) {
-			mods.push(this.owner);
+		if (this.data.owner) {
+			mods.push(this.data.owner);
 		}
 		if (this.currentCollaborators) {
 			if (this.collaboratorListCollapsed) {
-				mods.push(...this.currentCollaborators.slice(0, this.initialCollaboratorCount));
+				mods.push(
+					...this.currentCollaborators.slice(0, this.data.initialCollaboratorCount)
```

Comment:
Again, check reactivity. I think if you accepted a collaboration you wouldn't show in the list until after a refresh with this change.

Reply:
Pretty sure we never supported this?
I just tested this in master and it didn't seem to update the list when you click accept.

Reply:
i did add this though, and it works without changing the bit of code highlighted here.
I think Vue reactivity goes into nested data like this unless the entire instance is swapped out.
Since it's passed in as a prop, Vue handles it correctly.